web/docs/operators/package.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_custom_props:
+  operator:
+    source: true
+    transformation: false
+    sink: true
+---
+
+# package
+
+Manages the packages installed on a given node.
```

Comment:
The node isn't a parameter, so "on a given node" doesn't quite make sense. Maybe this?

```suggestion
Manages packages at a node.
```